app/experimenter/experiments/forms.py

```diff
@@ -773,7 +774,11 @@ class ExperimentReviewForm(
     )
     review_qa_requested = forms.BooleanField(
         required=False,
-        label="QA Requested",
+        label=mark_safe(
+            'QA <a href="'
+            'https://moz-pi-test.atlassian.net/servicedesk/customer/portal/9" '
```

Comment:
I think this should be a [setting](https://github.com/mozilla/experimenter/blob/22b94d6534ebcca1927cd5b56ad1f98f3031c344/app/experimenter/settings.py#L310-L315) instead. That seems to be the style used in experimenter for URLs, and I imagine it's pretty likely for this to change over time or between environments.

Reply:
I made a change in this vein... let me know what you think.